script.js

```diff
@@ -0,0 +1,144 @@
+const todoForm = document.querySelector('.inputForm');
+const todoInput = todoForm.querySelector('input');
+const todoList = document.querySelector('.toDos');
+const doneList = document.querySelector('.dones');
+
+const saveList = 'savedList';
+let savedList = [];
+
+const saveFinishedList = 'savedFinList';
+let savedFinList = [];
+
+loadList();
+
+// To get the input from the form by adding an
+// event listener for submit button
+todoForm.addEventListener('submit', function (e) {
```

Comment:
es6 문법을 활용해보시는게 어떨까요??
```suggestion
todoForm.addEventListener('submit', (e) => {
```